src/Orleans.CodeGenerator/CodeGenerator.cs

```diff
@@ -488,11 +488,16 @@ private bool ValidForKnownTypes(INamedTypeSymbol type)
                 case TypeKind.TypeParameter:
                 case TypeKind.Submission:
                 {
-                    if (this.log.IsEnabled(LogLevel.Trace)) this.log.LogTrace($"{nameof(ValidForKnownTypes)} skipping type {type} of kind {type.TypeKind}");
+                    // if (this.log.IsEnabled(LogLevel.Trace)) this.log.LogTrace($"{nameof(ValidForKnownTypes)} skipping type {type} of kind {type.TypeKind}");
```

Comment:
Why comment this line out?

Reply:
To match the other commented out lines, no real reason. I could delete them all or guard them using a different condition

Reply:
I think we either leave them uncommented and guarded by 
```csharp
if (this.log.IsEnabled(LogLevel.Trace))
```
or delete them if they aren't needed. We try not to leave commented out code in the codebase as a general rule.

Reply:
deleted